### PR TITLE
jan dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-january-2024.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+Title: iRODS Development Update: January 2024
+Date: 2024-01-15 10:00
```
Comment (trel) [inline content/posts/irods-development-update-january-2024.md:2]: let's go for tuesday 16th

File: content/posts/irods-development-update-january-2024.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+Last month, we announced the release of the NetCDF project for iRODS 4.3.1. We're now looking into **possibly** archiving this project because we feel there are better alternatives (e.g. Python). If you have questions about this, don't hestitate to reach out to us via email or any other channel.
+
+That wraps up packaging and plugins. Let's talk about clients.
+
+First up is the Python iRODS client (PRC). For those users working in a 4.3.1 environment, you'll be happy to know support for **pam_password** has been implemented. Improvements for TTL have also been implemented. A new release of the PRC is inching closer and closer, so stay tuned.
+
+The HTTP API is seeing steady improvements. Support for HTTPS for OpenID Connect is very close. There's been a number of general improvements across the board. For example, configuring OpenID Connect is becoming simpler, the data objects endpoing now supports streaming reads like the C++ REST API, and the server now validates the configuration on startup.
```
Comment (trel) [inline content/posts/irods-development-update-january-2024.md:30]: There's -> There have

Comment (trel) [reply content/posts/irods-development-update-january-2024.md:30]: endpoing -> endpoint

File: content/posts/irods-development-update-january-2024.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
```
Comment (trel) [inline content/posts/irods-development-update-january-2024.md:31]: need a mention of k8s testing?

**Review by trel — COMMENTED** [review]
read-only friday!

File: content/posts/irods-development-update-january-2024.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2024.md]: done

File: content/posts/irods-development-update-january-2024.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2024.md]: done

File: content/posts/irods-development-update-january-2024.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-january-2024.md:33]: Added some words. Not sure how I feel about them yet, but let me know what you think.

**Comment by trel** [comment]
k8s looks fine.

happy to publish as is - but you've got three days to tweak it / think on it.

**Comment by korydraughn** [comment]
Cool cool. It's good enough for me.

Squashing.

**Comment by korydraughn** [comment]
Squashed!

**Comment by trel** [comment]
will click tues am.

**Pull request merged**